zebra-consensus/src/transaction.rs

```diff
@@ -388,6 +427,7 @@ where
         unimplemented!("V5 transaction validation is not yet complete");
     }
 
+    /// Verifies if a V5 `transaction` is supported by the network `upgrade`.
     fn verify_v5_transaction_network_upgrade(
         transaction: &Transaction,
         upgrade: NetworkUpgrade,
```

Comment:
> the network `upgrade`

is confusing, because `upgrade` is a `NetworkUpgrade`, not a `Network`.

Here's one possible fix:
```suggestion
    /// Verifies if a V5 `transaction` is supported by `network_upgrade`.
    fn verify_v5_transaction_network_upgrade(
        transaction: &Transaction,
        network_upgrade: NetworkUpgrade,
```

We could also say something like:
> the supplied network upgrade

Reply:
Done in [d0ea656](https://github.com/ZcashFoundation/zebra/pull/2302/commits/d0ea656ab2d07884bbbd2364908a3cfc6a282c8b).